app/routes/de.ich-suche-redezeit.tsx

```diff
@@ -90,76 +55,53 @@ export default function SearchingCoach() {
           <Form replace>
             <fieldset className="mt-8">
               <legend className="mb-4 inline-block text-xl">
-                Nach Sprache filtern
+                Filter by language
               </legend>
               {languages.map((lang: string) => (
-                <label className="min-h-4 mr-1 mb-1 inline-block" key={lang}>
-                  <input
-                    className="peer sr-only"
-                    type="radio"
-                    name="lang"
-                    value={lang}
-                    defaultChecked={currentLang === lang}
-                  />
-                  <span className="inline-flex cursor-pointer items-center gap-1 rounded-md border px-2 py-1 peer-checked:bg-gray-500 peer-checked:text-white">
-                    <span>{lang}</span>
-                  </span>
-                </label>
+                <CoachFilterTag
+                  disabled={false}
+                  key={lang}
+                  value={lang}
+                  name="lang"
+                  defaultValue={currentLang === lang}
+                  type="radio"
+                >
+                  {lang}
+                </CoachFilterTag>
               ))}
             </fieldset>
             <fieldset className="mt-8">
               <legend className="mb-4 inline-block text-xl">
-                Nach Schlagwort filtern
+                Filter by tag
```

Comment:
Hey @HerrBertling, are we changing the labels to English in the german version for this component? 
We might rework it with i18n, but asking just in case

Reply:
Other than that it looks good to me! :)

Reply:
Ooops. I’ll keep it as is for now, okay? Then `i18n` can do its thing to solve this later on.